### ISSUE TITLE
45px of iframe is hidden due to parent styles

### ISSUE THREAD

**Issue by rhysburnie** [chiunhau/p5-manager#13]
Due to the 45px margin left of  iframe 45px of the canvas will not be visible if setting canvas full size.

In the following example you would expect to see a 50 x 50 px red square in the top right corner.
Instead you see a 5 x 50 px red rectangle.

```
function setup() {
  createCanvas(windowWidth, windowHeight);
}
function draw() {
  var squareSize = 50;
  fill('red');
  rect(width - squareSize, 0, squareSize, squareSize);
}
```

This could be fixed by the following setup:

`createCanvas(windowWidth - 45, windowHeight);`

But that is obviously not a good idea for many reasons.
The styles for this is hidden in the p5-manager server, I couldn't figure out where it was installed on my local machine.

**Comment by chiunhau** [comment]
Yes it's a bug. I've fixed it and will release later. Thank you!

**Comment by chiunhau** [comment]
fixed in 0.2.9
